pfcwd/main.py

```diff
@@ -194,9 +195,23 @@ def interval(poll_interval):
     configdb.connect()
     pfcwd_info = {}
     if poll_interval is not None:
+        pfcwd_table = configdb.get_table(CONFIG_DB_PFC_WD_TABLE_NAME)
+        entry_min = 3000
+        for entry in pfcwd_table:
+            if("Ethernet" not in entry):
+                continue
+            detection_time_entry_value = int(configdb.get_entry(CONFIG_DB_PFC_WD_TABLE_NAME, entry).get('detection_time'))
+            restoration_time_entry_value = int(configdb.get_entry(CONFIG_DB_PFC_WD_TABLE_NAME, entry).get('restoration_time'))
+            if ((detection_time_entry_value != None) and (detection_time_entry_value < entry_min)):
+                entry_min = detection_time_entry_value
+            if ((restoration_time_entry_value != None) and (restoration_time_entry_value < entry_min)):
+                entry_min = restoration_time_entry_value
+        if entry_min <= poll_interval:
+            print >> sys.stderr, "unable to use polling_interval = {}ms, value is bigger or equal to the minimum in PFCWD table".format(poll_interval)
```

Comment:
Can you make this msg more meaningful? just saying 'minimum' is hard for the user to figure out the cause of the failure

Reply:
@neethajohn 
please review and comment if this is clear enough